R/FilteredDataset.R

```diff
@@ -458,6 +487,10 @@ FilteredDataset <- R6::R6Class( # nolint
     metadata = NULL,
     hash = character(0),
 
+    # if this has length > 0 then only varnames in this vector
+    # can be filtered
+    .filterable_varnames = NULL,
```

Comment:
why `.xyz`? All of other _data_ fields do not have dot in its names so it's somewhat breaking a convention

Reply:
changed